src/main/java/com/example/draftspringboot/controller/DraftController.java

```diff
@@ -29,9 +25,12 @@ public DraftResponse getById(@PathVariable @Parameter(description = "Id of somet
         return draftService.getById(id);
     }
 
-    @GetMapping
-    public List<DraftResponse> getAll() {
-        return draftService.getAll();
+    @Operation(summary = "Get list of something with pagination and search by part of name")
+    @GetMapping()
+    public Page<DraftResponse> searchByPartName(
+            @Parameter(description = "index of page") @RequestParam(required = false) Integer index,
+            @Parameter(description = "number of elements on the page") @RequestParam Integer limit,
```

Comment:
I would create all parameters as a non required.

Reply:
Ok.
I have done this because I thought that I have to do only one parameter as a non mandatory.